### DISCUSSION TITLE
Digispark digikeyboard not working with some laptops/desktops

### DISCUSSION THREAD

**Discussion by Xypher2000** [ArminJo/DigistumpArduino#48] [Q&A] [answered]
I have been using digispark keyboard for 4 years to automate bios changes and imaging prep and other various processes. I have found that Dell higher end units with the newer GUI bios like precision laptops/desktops ignore the digispark completely. Also HP laptops will ignore and even lockup for 30seconds when one is inserted. 

Is there a way to get those devices to take commands from it?

lower end Dell's like latitudes with the non-GUI bios's work great with them.

**Comment by ArminJo** [comment, marked as answer]
Wrong repo. Use micronucleus https://github.com/micronucleus/micronucleus
